infra/modules/azure_function_app/locals.tf

```diff
@@ -45,4 +45,6 @@ locals {
   private_dns_zone = {
     resource_group_name = var.private_dns_zone_resource_group_name == null ? var.virtual_network.resource_group_name : var.private_dns_zone_resource_group_name
   }
+
+  fake = "fake value"
```

Comment:
what's this?

Reply:
I trick to trigger the pipeline while it is not into master yet